src/rust/engine/process_execution/src/lib.rs

```diff
@@ -12,7 +12,11 @@
 // It is often more clear to show that nothing is being moved.
 #![allow(clippy::match_ref_pats)]
 // Subjective style.
-#![allow(clippy::len_without_is_empty, clippy::redundant_field_names)]
+#![allow(
+  clippy::len_without_is_empty,
+  clippy::redundant_field_names,
+  clippy::too_many_arguments
+)]
```

Comment:
Would it be useful to check that this header exists in all of our rust files in the pre-commit hook?